content/docs/get-started/policy-as-code/best-practices.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Best Practices for Writing Policy Packs (Preview)
+linktitle: Best Practices
+weight: 4
+menu:
+  getstarted:
+    parent: Policy as Code
+    identifier: policy-as-code-best-practices
+
+aliases: ["/docs/quickstart/policy-as-code/bet-practices/"]
+---
+{{% pac-preview %}}
+
+## Naming Policies
+
+Each policy within a Policy Pack must have a unique name. The name must be between 1 and 100 characters and may contain letters, numbers, dashes (-), underscores (_) or periods(.).
+
+## Policy Assertions
+
+Policy assertions should be complete sentences, specify the resource that has violated the policy, and be written using an imperative tone. The table below provides some examples of policy assertions.
+
+| ✅                                                    | ❌                           |
+| -----------                                           | -----------                  |
+| "The RDS cluster must specify a node type."           | "Specify a node type."       |
+| "The RDS cluster must have audit logging enabled."    | “Enable audit logging.”      |
```

Comment:
Nit: Not that it really matters, but we have inconsistent quotes. Hugo will automatically turn the regular quotes into smart quotes.

```suggestion
| "The RDS cluster must have audit logging enabled."    | "Enable audit logging."      |
```